build.gradle

```diff
@@ -156,7 +156,7 @@ subprojects {
   }
 
   dependencies {
-    testCompile 'junit:junit:4.11', 'org.hamcrest:hamcrest-library:1.3', 'org.mockito:mockito-core:1.9.5'
+    testCompile 'junit:junit:4.11', 'org.hamcrest:hamcrest-library:1.3', 'org.mockito:mockito-core:1.9.5', 'org.slf4j:slf4j-simple:1.7.7'
```

Comment:
What I meant with my request was:
```
    testCompile 'junit:junit:4.11', 'org.hamcrest:hamcrest-library:1.3', 'org.mockito:mockito-core:1.9.5'
    testRuntime 'org.slf4j:slf4j-simple:1.7.7'
```
I tested it in IntelliJ and it is working fine - that is running tests with JUnit runner do proper logging - tried with `EhcacheManagerTest`.